### ISSUE TITLE
Missing document from corpus ArguAna

### ISSUE THREAD

**Issue by NourOM02** [beir-cellar/beir#172]
### Dataset
ArguAna

### Problem
In the qrels, there is one instance where for the given query `test-free-speech-debate-yfsdfkhbwu-con03a` the relevant document had the following id `test-free-speech-debate-yfsdfkhbwu-con03b`. However, if you check the corpus and look up for this id, there is no such a document.

**Comment by thakur-nandan** [comment]
Yes, this is possible. This is likely an issue with the original dataset: ArguAna. Unfortunately, we would need to contact the original dataset authors in this case.
